### ISSUE TITLE
1.54.0 gives "Unable to preventDefault inside passive event listener invocation." for touch events

### ISSUE THREAD

**Issue by yaustar** [playcanvas/engine#4329]
Project: https://playcanvas.com/editor/scene/474021
Launch the scene and open devtools to enable mobile view
Refresh the tab so that the touch events can be registered
Tap on the screen
Error comes up in console:

```
Unable to preventDefault inside passive event listener invocation.
DetectDoubleTap.onTouchStart	@	detect-double-tap.js…848-c5e69d340c85:57
fire	@	event-handler.js:204
_handleTouchStart	@	touch-device.js:65
</span></div>
```

Issue does not occur in 1.53

**Comment by willeastcott** [comment]
@erikdubbelboer Any objection to us backing out your PR?

**Comment by yaustar** [comment]
Confirmed it is due to https://github.com/playcanvas/engine/pull/4279 change

**Comment by erikdubbelboer** [comment]
> @erikdubbelboer Any objection to us backing out your PR?

No, lets revert it. Maybe keep it for touchmove as that's the event where it can have performance impact? Interesting that nobody complained about this before with the `mouseup` and `mousedown` events.

**Comment by yaustar** [comment]
> Interesting that nobody complained about this before with the mouseup and mousedown events.

On mobile, there is a bigger need to call preventDefault due to the browser native gestures such as zooming, bringing magnification etc. Mouse/desktop browsers don't have this problem

**Comment by willeastcott** [comment]
@slimbuck Can you oversee the backing out the aforementioned PR and push out a 1.54.1 please?

**Comment by slimbuck** [comment]
Yes. Will perhaps also include a fix for #4330.

**Comment by slimbuck** [comment]
I reverted the change and can confirm it fixes the touch errors.

https://github.com/playcanvas/engine/commits/release-1.54

**Comment by slimbuck** [comment]
This change will be published shortly in engine v1.54.1.
